compose.yaml

```diff
@@ -3,6 +3,9 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.dev
+    volumes:
+      - .:/usr/src/app
+      - /usr/src/app/node_modules
```

Comment:
please do not use `main` in your fork, because these changes should not be in your 2nd PR

Reply:
So i should checkout to some other branch and then push the changes.Right?

Reply:
You should always create a new branch from `main`, but you never push to main, your changes only enter main when your PR is accepted and merged, then you update your main to be in sync with upstream

I have a video demoing and explain on my youtube channel